.github/workflows/publish.yml

```diff
@@ -0,0 +1,50 @@
+name: Create a Docker image and publish it to GHCR
+
+on:
+  push:
+    #branches: ['release']
+    branches: ['main', 'feat/container']
```

Comment:
TODO

Reply:
![image](https://github.com/user-attachments/assets/2e66a5af-1352-42db-8d1a-76557989c796)
it works though